src/build.sh

```diff
@@ -301,6 +301,8 @@ for branch in ${BRANCH_NAME//,/ }; do
             sha256sum "$build" > "$ZIP_DIR/$zipsubdir/$build.sha256sum"
           done
           find . -maxdepth 1 -name 'lineage-*.zip*' -type f -exec mv {} "$ZIP_DIR/$zipsubdir/" \; &>> "$DEBUG_LOG"
+	  recovery_name=$(echo "$build" | sed "s/.zip//")
```

Comment:
Please dont reuse the iteration variable from for loop, but rather build the name from scratch using build_date and codename.

Reply:
done. thanks for the hint